src/freeze-dry/inline-images.test.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-env jest */
+/* eslint import/namespace: "off" */
+
+import inlineImages from 'src/freeze-dry/inline-images'
+import * as whenAllSettled from 'src/util/when-all-settled'
+import * as common from 'src/freeze-dry/common'
+
+beforeAll(() => {
+    common.inlineUrlsInAttributes = jest.fn()
+})
+
+describe('inlineImages', () => {
+    test('should call whenAllSettled with the jobs', async () => {
+        const spy = jest.spyOn(whenAllSettled, 'default')
+        const rootElement = window.document.implementation.createHTMLDocument()
+        const docUrl = 'about:blank'
+        const jobs = [
+            common.inlineUrlsInAttributes({...{}, rootElement, docUrl}),
+            common.inlineUrlsInAttributes({...{}, rootElement, docUrl}),
+            common.inlineUrlsInAttributes({...{}, rootElement, docUrl}),
+        ]
+        await inlineImages({rootElement, docUrl})
+        expect(spy).toHaveBeenCalledWith(jobs)
```

Comment:
__Shivangs question:__

> 3:53] 
Your review at this line saying this doesn't test anything and you would rather test if the rootElement be modified correctly

> [3:54] 
I just had a small doubt, since these are unit tests, isn't it the gist of the unit test to make sure the code is executed in order without using external resources. If I made it such that I check for the modification of the rootElement then that would classify as an integrations test

> [3:55] 
Also it would require me to unmock the inlineUrlsInAttributes function as well

> [3:57] 
I'm not too sure of this, but in one of the best practices it was stated a unit tests should run comletely independent from rest of the code. Such that any change to external code shouldn't affect that particular test until changes are made to the module on which it is tested. In this case, if we change the inlineUrlsInAtrribute function, we will have to change the unit tests for this module as well

__My answer:__

Ok, so inlineUrlsInAttributes is tested in common.test.js which is fine. So forget about the " I would rather test if the rootElement has been modified correctly by comparing the result with the expected DOM tree.".

About the "I think this doesn't really test anything." I would suggest to expect that common.inlineUrlsInAttributes is called correctly.